include/nccl_ofi_rdma.h

```diff
@@ -730,12 +734,12 @@ struct nccl_net_ofi_rdma_ep {
 	/* Pending requests queue */
 	nccl_ofi_deque_t *pending_reqs_queue;
 
-	/* Free list of bounce buffers */
-	nccl_ofi_freelist_t *bounce_buff_fl;
+	/* Free list of bounce buffers (ctrl rail) */
+	nccl_ofi_freelist_t *bounce_buff_ctrl_fl;
+	/* Free list of bounce buffers (data rails) */
+	nccl_ofi_freelist_t *bounce_buff_data_fl;
```

Comment:
why are we not splitting requests at the same time?

Reply:
The freelists for ctrl recv requests and eager recv requests will be identical, so splitting them doesn't accomplish anything functionality-wise.

That being said, splitting the freelists might make the code more readable if we also split the eager buffer freelists by rail -- then we could move these freelists to being owned by rails, as discussed in [comment](https://github.com/aws/aws-ofi-nccl/pull/614/files#r1767701208).

Reply:
Split.